R/derive_vars_aage.R

```diff
@@ -123,22 +123,64 @@ NULL
 #' data.frame(AGE = 1:100) %>%
 #'   derive_agegr_fda(age_var = AGE, new_var = AGEGR1)
 #'
-derive_agegr_fda <- function(dataset, age_var, new_var) {
+derive_agegr_fda <- function(dataset, age_var, age_unit = NULL, new_var) {
 
   age_var <- assert_symbol(enquo(age_var))
   new_var <- assert_symbol(enquo(new_var))
   assert_data_frame(dataset, required_vars = quo_c(age_var))
```

Comment:
Should we check the type of the `age_var` column in the input dataset? If `age_var` refers to a character variable the error message is not very helpful,